### PR TITLE
Moved manual install notice to proposal UI

### PR DESCRIPTION
**Pull request by tboerger**
suse-manager-client

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
:+1:

**Comment by rsalevsky** [comment]
+1

**Comment by tboerger** [comment]
So just waiting for a confirmation by @vuntz or maybe even @aspiers as he's a native speaker :)

File: crowbar_framework/config/locales/suse_manager_client/en.yml [hunk 1]
```diff
@@ -22,3 +22,12 @@ en:
         manager_server_hint: 'For example susemgr.example.com. You can set a record within the %{link}.'
         activation_key: 'Activation Key'
         activation_key_hint: 'For example 1-00112233445566778899aabbccddeeff'
+
+        manual_steps: |
+          In order to use this barclamp you have to prepare some manual steps.
```
Comment (vuntz) [inline crowbar_framework/config/locales/suse_manager_client/en.yml]: Add `,` after "this barclamp".

Comment (vuntz) [inline crowbar_framework/config/locales/suse_manager_client/en.yml]: And maybe s/prepare/execute/

File: crowbar_framework/config/locales/suse_manager_client/en.yml [hunk 2]
```diff
@@ -22,3 +22,12 @@ en:
...
+          After installing the SUSE Manager Server you have to download the RPM
+          from <strong>%{manager_url}</strong> (check the version number) and
```
Comment (vuntz) [inline crowbar_framework/config/locales/suse_manager_client/en.yml]: I know it was like that before, but I have no clue about why `check the version number`. Maybe we should just drop that bit if we can't explain it?

File: crowbar_framework/config/locales/suse_manager_client/en.yml [hunk 3]
```diff
@@ -22,3 +22,12 @@ en:
...
+          place it at <strong>%{rpm_path}</strong>. To finish the preparation
+          you have to execute <strong>%{install_command}</strong> on the Admin
+          node. Now you can apply the SUSE Manager Client proposal on the
```
Comment (vuntz) [inline crowbar_framework/config/locales/suse_manager_client/en.yml]: Administration Server, not Admin node.

File: crowbar_framework/config/locales/suse_manager_client/en.yml [hunk 4]
```diff
@@ -22,3 +22,12 @@ en:
...
+          selected nodes.
```
Comment (vuntz) [inline crowbar_framework/config/locales/suse_manager_client/en.yml]: Last sentence can be dropped, imho.

**Comment by tboerger** [comment]
@vuntz I have integrated your comments.

File: crowbar_framework/config/locales/suse_manager_client/en.yml [hunk 5]
```diff
@@ -22,3 +22,11 @@ en:
...
         activation_key: 'Activation Key'
         activation_key_hint: 'For example 1-00112233445566778899aabbccddeeff'
+
+        manual_steps: |
+          In order to use this barclamp, you have to execute some manual
+          steps. After installing the SUSE Manager Server you have to
+          download the RPM from <strong>%{manager_url}</strong> and place
+          it at <strong>%{rpm_path}</strong>. To finish the preparation you
+          have to execute <strong>%{install_command}</strong> on the
+          Administration Server.
```
Comment (vuntz) [inline crowbar_framework/config/locales/suse_manager_client/en.yml]: Bah, not sure how I missed them, but I'd still add some commas and rephrase further. I'd also remove the "you", so let me rephrase this to:
```
Manual steps are required in order to use this barclamp. After
installing the SUSE Manager Server, download
<strong>%{manager_url}</strong> and place the file at
<strong>%{rpm_path}</strong>. To finish the setup, execute
<strong>%{install_command}</strong> on the Administration
Server.
```

Comment (vuntz) [inline crowbar_framework/config/locales/suse_manager_client/en.yml]: +1 after this change, btw.

**Comment by tboerger** [comment]
@vuntz done, fine to merge?

**Comment by vuntz** [comment]
I gave my +1 already in my previous comment, but here it is again :-)
